jekyll/_cci2/writing-yaml.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: classic-docs
+title: "Writing YAML"
+short-title: "Writing YAML"
+description: "How to Write YAML on CircleCI"
+order: 20
```

Comment:
there should be a category, see template in wiki

Reply:
I added a note to the New Doc Template about where to find the available categories.